src/layouts/partials/Header.vue

```diff
@@ -35,7 +35,7 @@
 
         <div class="has-dropdown">
           <a
-            aria-label="github"
+            aria-label="GitHub"
```

Comment:
For human-readable prose, I agree with the official casing, but for HTML elements and attributes, I'd rather just keep it simple and consistent.

Reply:
Supposedly, these will be read back the same way regardless of which we go with (which may be true for some screen readers), however I just tested this with the Windows 10 Narrator and it isn't pronounced correctly without the capitals in both spots (`GitHub`). 

At a higher level, I would expect aria-label to follow the same guidance as any sentence visually displayed to end users since it functions as a descriptive sentence that will be read to users using assistive technologies.

Reply:
Oh, I understand. I thought this was purely a symbolic identifier. If it's human-facing, you're right that it should have proper casing. Thanks!